deployments/ansible/roles/collector/README.md

```diff
@@ -63,20 +75,25 @@ how to use the role in a playbook with minimal required configuration:
   config, e.g. `./custom_collector_config.yaml`. (**default:** `""` meaning 
   that nothing will be copied and existing `splunk_otel_collector_config` will be used)
 
-- `splunk_bundle_dir`: The path to the [Smart Agent bundle directory](
+- `collector_path_win`: Default path of Splunk-otel-collector in windows 
+  (**default:** `C:\\Program Files\Splunk\OpenTelemetry Collector\`)
+
+- `splunk_bundle_dir` & : The path to the [Smart Agent bundle directory](
   https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/README.md).
   The default path is provided by the collector package. If the specified path
   is changed from the default value, the path should be an existing directory
   on the node. The `SPLUNK_BUNDLE_DIR` environment variable will be set to
   this value for the collector service.  (**default:**
-  `/usr/lib/splunk-otel-collector/agent-bundle`)
+  `/usr/lib/splunk-otel-collector/agent-bundle` For Windows `${collector_path_win}\agent-bundle`)
```

Comment:
This Linux versus Windows thing is hard to read -- perhaps all these options can be turned into a table

Parameter (default) | OS | Description

Where OS is Linux|Windows|All

Reply:
Another alternative is to have different sections per OS -- this would result in duplicated information, but perhaps fine as it make it more clear

Reply:
Sure will update in that format.